src/app/static/src/app/components/plots/table/Table.vue

```diff
@@ -2,23 +2,34 @@
     <div>
         <br>
         <div v-if="filteredData.length > 0">
-            <table class="table">
-                <div>
-                    <tr>
-                        <th v-translate="'area'"></th>
-                        <th v-for="f in filtersToDisplay" v-translate="f.label"></th>
-                        <th v-for="i in indicators">{{ i.name }}</th>
-                    </tr>
-                    <tr v-for="row in filteredData">
-                        <td>
-                          <div>{{ row.areaLabel }}</div>
-                          <div class="small">{{ row.areaHierarchy }}</div>
-                        </td>
-                        <td v-for="f in filtersToDisplay">{{ row.filterLabels[f.id] }}</td>
-                        <td v-for="i in indicators">{{ row.indicatorValues[i.indicator] }}</td>
-                    </tr>
-                </div>
-            </table>
+            <b-form-group
+              class="mb-0"
+            >
+              <b-input-group size="sm">
+                <b-form-input
+                  v-model="filter"
+                  type="search"
+                  id="filterInput"
+                  placeholder="Type to Search"
+                ></b-form-input>
+                <b-input-group-append>
+                  <b-button :disabled="!filter" @click="filter = ''">Clear</b-button>
```

Comment:
Could do with a test for this "clear" functionality

Reply:
Added

Reply:
Cool, I've formatted it for you here but we should get you set up with IntelliJ or sort out code formatting rules for VS when you get the new laptop set up. We should also set up code style checking as part of CI rather than relying on PRs to pick up formatting issues, been meaning to do that for a while!